src/UI/Component/Button/Factory.php

```diff
@@ -260,4 +260,38 @@ public function tag($label, $action);
 	 */
 	public function bulky($icon_or_glyph, $label, $action);
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *       The Toggle Button triggers the activation/deactivation of some content already shown on the screen, i.e.
+	 *       a filter. The deactivation of content means, that it is still shown and the user can still interact
+	 *       with it, but it has no effect to the system.
+	 *   composition: >
+	 *       The Toggle Button uses the primary color as background when it is activated.
+	 *       The Toggle Button uses the primary color with a grayscale filter as background when it is deactivated.
+	 *   effect: >
+	 *       Clicking the Toggle Button activates/deactivates the related content. The activated/deactivated state of
+	 *       the content is visually noticeable for the user, i.e. by greying out the content in the deactivated state
+	 *       (like the Toggle Button itself).
```

Comment:
We would not want to use a toggle button to just hide stuff, right? This imho is what the expander glyph is used for, so we should add it here as rival.

Reply:
Done.